lib/perl/OVH/Bastion.pm

```diff
@@ -761,16 +762,69 @@ sub is_valid_remote_user {
     return R('ERR_INVALID_PARAMETER', msg => "Specified user doesn't seem to be valid");
 }
 
+sub validate_proxy_params {
+    my %params         = @_;
+    my $proxyHost      = $params{'proxyHost'};
+    my $proxyPort      = $params{'proxyPort'};
+    my $proxyUser      = $params{'proxyUser'};
+    my $allowWildcards = $params{'allowWildcards'} // 1;
+
+    my $proxyIp;
+    my $fnret;
+
+    if ($proxyHost) {
+        $fnret = OVH::Bastion::is_valid_ip(ip => $proxyHost, allowSubnets => 0);
+        if ($fnret) {
+            $proxyIp = $fnret->value->{'ip'};
+        }
+        else {
+            $fnret = OVH::Bastion::get_ip(host => $proxyHost);
+            if ($fnret) {
+                $proxyIp = $fnret->value->{'ip'};
+            }
+            else {
+                $proxyIp = $proxyHost;
```

Comment:
I would have expected an error return because the host can't be resolved, is there a case I'm missing where this would be needed?